packages/react-components/src/components/Picker/PickerList.module.scss

```diff
@@ -114,15 +114,50 @@ $base-class: 'picker-list';
     }
 
     &__content {
+      display: flex;
       flex: 1;
+      align-items: center;
       overflow: hidden;
-      text-overflow: ellipsis;
-      white-space: nowrap;
     }
 
     &__custom {
       width: 100%;
       height: auto;
     }
+
+    &__checkbox {
+      align-self: center;
+      margin-right: var(--spacing-2);
+    }
+
+    &__avatar,
+    &__icon {
+      align-self: center;
+      margin-right: var(--spacing-2);
+      border-radius: var(--radius-3);
+      width: 20px;
+      height: 20px;
+    }
+
+    &__label-container {
+      display: flex;
+      flex: 1;
+      flex-direction: column;
+      justify-content: center;
+      overflow: hidden;
+    }
+
+    &--two-line {
+      height: 52px;
+    }
+
+    &__main-label {
+      font-weight: bold;
+    }
+
+    &__secondary-label {
+      color: var(--content-basic-secondary);
+      font-size: 0.8em;
```

Comment:
IMO, we should stick to value types that we are already using in the project - pixels unless we are planning to switch our library `font-size` properties to the `em` values.

Reply:
fixed here: https://github.com/livechat/design-system/pull/741/commits/635dfcd0246ada4d178c3f055fad2d534273f666